### PR TITLE
Extend cooking generators with dishes/meals, starters, main courses and desserts

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR adds value generators for dishes/meals, starters, main courses and desserts.

Note that to get things to build, I had to submit another PR (#56). This PR is thus dependent on #56.

### CODE REVIEW

File: src/GenFu/Resources/CookingDesserts.txt [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+Tiramisu
+Treacle tart
+Trifle
+Twinkie
+Waffles
+Waldorf pudding
+Watergate salad
+Whoopie pie
+Zabaione
+
```
Comment (MisterJames) [inline src/GenFu/Resources/CookingDesserts.txt]: Can you nuke the empty lines here?

**Comment by MisterJames** [comment]
So...what is the case for meal? I like desserts, mains and starters, but you have to sell me on the meal.

**Comment by M-Zuber** [comment]
Restaurants. At least here a lot have a separate concept of meals lunch time for business people

**Comment by M-Zuber** [comment]
Take out places. 
And I know someone who works for a company producing software for both these areas

**Comment by ErikSchierboom** [comment]
What @M-Zuber says is basically the idea. Do you want me to remove it?

**Comment by MisterJames** [comment]
@stimms @dpaquette what are your thoughts? I think the composite field could be done easily with a custom property filler lambda, not sure we need the all-in-one string together.

**Pull request closed without merging**